CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog - chai-webdriverio
+
+## Unreleased Changes
+#### Breaking Changes:
+- Changed `there` and `visible` from methods to properties:
```

Comment:
Wouldn't that fail because visible is undefined without the plugin?  I guess I'm fine with either way if there's a real issue with this.  I was mostly changing it for consistency :)  So if you decide not to change these, you can just close this one. (And we can deal with immediately in another PR once we figure it out)